src/views/admin/applications/AdminHome.tsx

```diff
@@ -0,0 +1,161 @@
+import axios from "axios";
+import { TitleBanner } from "components";
+import Router from "next/router";
+import { useContext, useEffect, useState } from "react";
+import { Col, Row } from "react-bootstrap";
+import UserContext from "utils/userContext";
+import Application from "../components/Application/Application";
+import ApplicationSearch from "../components/ApplicationSearch/ApplicationSearch";
+import styles from "./AdminHome.module.scss";
+
+// Applicant stores an applicant's responses
+export interface Applicant {
+	first_name: string;
+	last_name: string;
+	email: string;
+	gender: string;
+	pronouns: Array<string>;
+	ethnicity: string;
+	is_18_older: boolean;
+	university: string;
+	education_level: string;
+	major: string;
+	is_first_hackathon: boolean;
+	portfolio_link: string | null;
+	linkedin_link: string | null;
+	stress_relief_question: string;
+	company_specialize_question: string;
+	resume_url: string;
+	submission_time: string;
+}
+
+// SidebarDisplay stores an applicant's name and string. This will be used
+// to display search results for applicants (formatted as name and status)
+// in the admin portal.
+export interface SidebarDisplay {
+	name: string;
+	status: Status;
+}
+
+export enum Status {
+	accepted,
+	rejected,
+	waitlisted,
+	pending,
+}
+
+interface User {
+	_id: string;
+	guest_auth?: object;
+	application_data: Applicant;
+	role: string;
+	status: string;
+}
+
+function AdminHome() {
+	const { uid, role } = useContext(UserContext);
+	const isLoggedIn = uid !== null;
+	const hasAccess = role === "admin";
+
+	const [userDisplayed, setUserDisplayed] = useState<number>(0);
+	const [sidebarInfo, setSidebarInfo] = useState<SidebarDisplay[]>([
+		{
+			name: "Peter Anteater",
+			status: Status.pending,
+		},
+	]);
```

Comment:
Suggestion: having a derived state may make the data more difficult to manage. We could compute this data based on a primary reference rather than storing it in state.